package.json

```diff
@@ -22,6 +22,10 @@
     "deploy:docs": "node ./scripts/deploy-docs.js",
     "deploy:package": "scripts/publish.sh",
     "commitmsg": "commitlint -e $GIT_PARAMS",
+    "contributors:add": "all-contributors add",
+    "contributors:check": "all-contributors check",
+    "contributors:generate": "all-contributors generate",
```

Comment:
I see that `yarn contributors:update` is used in the precommit. But what about the other ones? When do you use add, check, and generate? Do we need to run these at some other time? If not, then perhaps we don't need these extra yarn scripts?

Reply:
I put those in there just in case we would want to run things manually for debugging purposes, but you're right, we can probably just remove these and use `npx` or something.